### DISCUSSION TITLE
Disable macOS cursor shake feature with osu! open

### DISCUSSION THREAD

**Discussion by NiceAesth** [ppy/osu#19589] [Ideas] [answered]
macOS has a feature where you can shake the cursor in order for it to be enlarged so you can find it. This happens quite often with osu! open and it would be very helpful if it could be prevented while the game was open.

**Comment by nagi-desuuu** [comment]
Surely it can be disabled in the macOS settings entirely, right?

**Reply by NiceAesth** [reply]
Doesn't really fix the problem

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/issues/11772

**Reply by NiceAesth** [reply]
Ah, my bad. Thought it wouldn't count as an issue so I only checked discussions.
